test/domains/vass/domain-delete.chpl

```diff
@@ -0,0 +1,4 @@
+// formerly BenA's types/records/compilerErrors/deleteRecordMessage
```

Comment:
arrays and domains are records. Anyway, I don't think it's *too* important. I'm fine with domain- & array- specific paths, but not vass-specific paths 😛